openpower/package/openpower-pnor/openpower-pnor.mk

```diff
@@ -8,7 +8,7 @@
 # make doesn't care for quotes in the dependencies.
 XML_PACKAGE=$(subst $\",,$(BR2_OPENPOWER_XML_PACKAGE))
 
-OPENPOWER_PNOR_VERSION ?= 1e4410052a26acf4b439226860afae9507f0d94a
+OPENPOWER_PNOR_VERSION ?= f5b523fb0a7fe312c5d011e9853f53c22a2f2262
```

Comment:
Minor, but a rebase should pick this version up already.  I believe the jenkins cronjob got it in last night.